### DISCUSSION TITLE
Why cant we hide manu bar icon?

### DISCUSSION THREAD

**Discussion by sontiO** [will-stone/browserosaurus#676] [General]
I dont see the point of forcing the menu bar icon. It has very little use/never gets used by me. Is there any good reason theres no "show/hide" menubar option in the preferences?

**Comment by mariozelaschi** [comment]
@will-stone 
Hi, and thanks so much for this super useful (and free!) software!
I also would love to have the option to hide the menu icon.
I know there are softwares to hide menu bar icons, but I try not to install too many utils. Space in the menu bar is very limited because of the notch and would be nice to have this option. For instance, Alt Tab opens the preferences window when the icon is off by relaunching the app when it is already running.

PS even maybe an option to write to the plist from terminal would be amazing, eg defaults write com.xxx.xxx hideIcon -bool

Just a suggestion :)
Thanks!

**Reply by will-stone** [reply]
Just wait for Tahoe 🤓
